ompi/tools/mpirun/main.c

```diff
@@ -28,46 +28,10 @@
 #include "opal/util/printf.h"
 #include "opal/util/show_help.h"
 #include "ompi/constants.h"
-
-static char *find_prterun(void)
-{
-    char *filename = NULL;
-#if !OMPI_USING_INTERNAL_PRRTE
-    char *prrte_prefix = NULL;
-#endif
-
-    /* 1) Did the user tell us exactly where to find prterun? */
-    filename = getenv("OMPI_PRTERUN");
-    if (NULL != filename) {
-        return filename;
-    }
-
 #if OMPI_USING_INTERNAL_PRRTE
-    /* 2) If using internal PRRTE, use our bindir.  Note that this
-     * will obey OPAL_PREFIX and OPAL_DESTDIR */
-    opal_asprintf(&filename, "%s%sprterun", opal_install_dirs.bindir, OPAL_PATH_SEP);
-    return filename;
-#else
-
-    /* 3) Look in ${PRTE_PREFIX}/bin */
-    prrte_prefix = getenv("PRTE_PREFIX");
-    if (NULL != prrte_prefix) {
-        opal_asprintf(&filename, "%s%sbin%sprterun", prrte_prefix, OPAL_PATH_SEP, OPAL_PATH_SEP);
-        return filename;
-    }
-
-    /* 4) See if configure told us where to look, if set */
-#if defined(OMPI_PRTERUN_PATH)
-    return strdup(OMPI_PRTERUN_PATH);
-#else
-
-    /* 5) Use path search */
-    filename = opal_find_absolute_path("prterun");
-
-    return filename;
-#endif
+#include "3rd-party/prrte/include/prte.h"
```

Comment:
Are we only supporting `mpirun` with the internal prte?  If so, why does the configury test for `prte_launch` with an external prte?